diff --git a/Concentration-Core/Card.java b/Concentration-Core/Card.java
index 41b6221..d1934d8 100644
--- a/Concentration-Core/Card.java
+++ b/Concentration-Core/Card.java
@@ -23,17 +23,21 @@ public class Card {
      *   new Card(Color.BLUE)
      * and it returns a new "instance" of the Card class.
      */
-    public Card(Color color) {
+    public Card(Color requestedColor) {
         // Fields can either be initialized where they're declared, like isFaceUp and isMatched, or they can be
         // initialized in class constructors.
-        this.color = color;
+
+        // Currently, we are setting the color of every card to the constant value of Color.PINK. Fix the code by
+        // setting color to the value of the requested color.
+        color = Color.PINK;
     }
 
     /**
      * This method is responsible for noting when a card has been matched.
      */
     void setMatched() {
-        isMatched = true;
+        // Currently, nothing happens when this method is called. Fix the code by updating the value of an appropriate
+        // field in the Card class.
     }
 
     /**
@@ -53,6 +57,11 @@ public class Card {
      * is what happens when you flip a card in a card game.
      */
     void flip() {
-        isFaceUp = !isFaceUp;
+        // EXTRA CREDIT: can you rewrite this code to have the same functionality, but only use one line of code?
+        if (isFaceUp) {
+            isFaceUp = false;
+        } else {
+            isFaceUp = true;
+        }
     }
 }
